index.js

```diff
@@ -166,6 +149,10 @@ module.exports = function () {
       'dropped6': Buffer.alloc(0)
     })
   }
+}
+
+utPex.prototype.name = 'ut_pex'
 
+module.exports = () => {
```

Comment:
I think this should export the class because bittorrent-protocol uses the new operator.

Reply:
ok, are you sure?

webtorrent uses this module like so:

```js
    // use ut_pex extension if the torrent is not flagged as private
    if (typeof utPex === 'function' && !this.private) {
      wire.use(utPex())
```

Reply:
No I'm not sure. That seems right if that's how webtorrent uses this. 

Seems unintuitive and unnecessary though. 

Ah, you talk about it here https://github.com/webtorrent/ut_metadata/pull/41